server/src/main/java/org/elasticsearch/index/mapper/ObjectMapper.java

```diff
@@ -569,4 +569,5 @@ void toXContent(XContentBuilder builder, Params params, ToXContent custom) throw
     protected void doXContent(XContentBuilder builder, Params params) throws IOException {
 
     }
+
```

Comment:
Maybe remove this from the PR just so I don't have to keep scrolling by it....

Reply:
die, random newline, die!